articles/deploying-nodejs-web-app/index.md

```diff
@@ -0,0 +1,143 @@
+
+**Add header image**
+
+Congratulations, you’ve finished developing your first NodeJS web app and now you want to publish it on the web. If you’re still in the development process however, then you may find my **(Link to previous developing NodeJS web app when it gets published)** guide helpful.
+
+There are many hosting platforms you can use to deploy your NodeJS web apps such as [Heroku](https://www.heroku.com), [Vultr](https://www.vultr.com), [Linode](https://www.linode.com), [Google Cloud Platform](https://console.cloud.google.com) and [Amazon Web Services](https://aws.amazon.com). We will be using [DigitalOcean](https://www.digitalocean.com) because it’s very popular, simple to use and good value.
+
+### Setting up DigitalOcean
+
+First, create an account on the DigitalOcean platform. There are discount codes available to add free credit to your account such as the code available in the Github Student Developer Pack. Be aware that you can only redeem one code per account.
+
+Second, you need to create a droplet. A droplet is a VPS (Virtual Private Server.) It’s similar to a Linux VM which is hosted on a server farm somewhere. Once you’ve logged into your account, go to Droplets under the Manage heading and click Create and Droplets.
+
+![DigitalOcean Droplet Creation](/engineering-education/deploying-nodejs-web-app/digitalocean-creation.png)
+
+You can leave most of the settings as the default but change the plan to the basic $5 a month which contains enough resources for your app. You can scale this up later if needed.
+
+Also, choose the datacenter closest to the target audience of your app and change the authentication to password. While password authentication is less secure (SSH Keys is recommended), it’s much easier to set up so for your first drop we’ll use that.
+
+All that’s left now is to pick a name (hostname) and click Create Droplet.
+
+### Connecting to your Droplet
+
+Shortly afterward, you’ll receive an email containing the username and password of your droplet which you’ll use to login.
+
+Back on the DigitalOcean website, under Droplets, click the name of your newly created droplet and then on Console. This will open a new tab that will let you control your droplet. Alternatively, you can use any SSH client with the IP address and user credentials contained in the email.
+
+On your first login, since you used password authentication, it will prompt you to set a new password. A great way to generate secure passwords and store them is a password manager like [LastPass](https://www.lastpass.com).
+
+### Deploying Your NodeJS Web App
+First, you’ll need to copy the code for your web app to your Droplet. If you’re using source control such as Git **Link to Section article on Git when it's published** then it’s a simple as installing git using `apt-get install git -y`  and then using the git clone command  `git clone (link to your repository)` and then add the link to your repository at the end.
```

Comment:
You switch back and forth a bit from capitalizing droplet and not. I would pick one and stick with it.